src/Xamarin.Forms.StateSquid.Multi/Platforms/Shared/State.cs

```diff
@@ -8,6 +8,10 @@ public enum State
         Success,
         Error,
         Empty,
-        Custom
+        Custom,
```

Comment:
I don't necessarily want to add all sorts of arbitrary states, because it's hard to draw a line in the sand when adding these. They are essentially all different types of the existing `Error` state which could all be done using `Custom`. So I'm not looking to change the current set of states at this time.